### ISSUE TITLE
A mention of Hacktoberfest?

### ISSUE THREAD

**Issue by swedishmike** [DinisCruz/Book_Generation_Z_Developer#30]
I just had a thought that a mention of Hacktoberfest, https://hacktoberfest.digitalocean.com/ , might be useful?

It's a good way of getting to learn how to use Git(hub) and a good way of getting into supporting Open Source projects.

Also, who can't find use for another (free) t-shirt and some more (again free) laptop stickers? ;)

**Comment by DinisCruz** [comment]
Yeah, that is a cool idea
![image](https://user-images.githubusercontent.com/656739/46583520-96bd8f80-ca50-11e8-85a0-fb2d9857ebc3.png)

That only lasts this month right?

**Comment by swedishmike** [comment]
Yeah - it lasts for one month. Register, submit five PR's and one the month is up you get a t-shirt and some stickers. A good way of 'bribing' people to look at projects and give assistance. 

Some projects have done quite well out of tagging them with hacktoberfest too.

**Comment by DinisCruz** [comment]
I wonder if I can add this book to that list :)
